CONTRIBUTING.md

```diff
@@ -1,30 +1,48 @@
-# Contributing to Devicon
-
-First of all, thanks for taking the time to contribute!
-
-These are not rules, just guidelines that you can follow in order to keep the repository as clean as possible. But if you don't have time to follow all of it, no worries, just create a pull request and I'll make the modifications myself if needed.
-
-And if you think there is a better way to organize this repo, please do not hesitate to share your point of view.
-
-## Submitting icon(s)
-
-Each icon comes in different "versions":
-- original
-- original-wordmark
-- plain
-- plain-wordmark
-- line
-- line-wordmark
-
-This is not mandatory, an icon can only have 1 or 2 versions available. Just keep in mind that the minimum is 1 and the maximum 6 (for now).
-
-The plain and line versions (with or without wordmark) are designed to be available in the final icon font. So they need to stay as simple as possible (one color and ensure that the paths are united before to export to svg). You can use a service like [compressor](https://compressor.io/) or [SVG Editor](https://petercollingridge.appspot.com/svg-editor) in order to optimize the svg file.
```

Comment:
Or [SVOMG](https://svgomg.firebaseapp.com/) or SVGO.

Reply:
Your link to https://svgomg.firebaseapp.com/ throws a 404 not found

Reply:
Really? it works on my hand.
![Uploading image.png…]()

Reply:
When I enter https://svgomg.firebaseapp.com/ i get the following response:
![image](https://user-images.githubusercontent.com/8781699/84131754-44aee780-aa45-11ea-82c2-7f0b27017530.png)

What about https://jakearchibald.github.io/svgomg/? is that the page you were looking for?

Reply:
That's bizarre, I found that on DuckDuckGo and that same link works on my hand.

> What about https://jakearchibald.github.io/svgomg/? is that the page you were looking for?

Seems like the same tool so yeah.